### DISCUSSION TITLE
Miscellaneous improvement to track recording

### DISCUSSION THREAD

**Discussion by Berfur** [andreynovikov/trekarta#173] [Ideas]
Hi,
  I am happily using Trekarta to map my mountain walks. I'd like to suggest some (I think easy) improvements:

* Pause: is it possible to add the possibility to pause the recording? I usually record my ascension, then I stop in some place to eat and then record the return. But I usually have to record two different tracks. So if it's possible I'd like to have the possibility to pause, not stop the recording.
* Note: is it possible to add notations to the track? Like a text notation?
* Add in the statistics the Cumulative Elevation Gain (https://en.wikipedia.org/wiki/Cumulative_elevation_gain)

Thank you!

**Comment by andreynovikov** [comment]
Currently you can pause it by leaving application and pressing Pause in notification. I looks like a good option to add this inside application.

**Comment by andreynovikov** [comment]
Added pausing and statistics. Will think about notes. It's not obvious.

**Comment by andreynovikov** [comment]
By note do you mean just a description to hole track or notes to specific points along track?
